### PR TITLE
Gracefully handle unsupported voxel types

### LINKED ISSUE #3608: WK fails silently when unsupported wkw data type is used
For example, if wkw was created with `np.int32` (instead of `np.uint32`) buckets are just "not found". See here, for a complete list of supported wkw data types: https://github.com/scalableminds/webknossos-wrap/blob/f161f97248279d262081b8f428b8a6f4fa393cf7/rust/src/header.rs#L254-L254

Not sure whether this change would need to be implemented in the wkw lib itself. @fm3 Do you know whether this is a back-end change?

### PR DESCRIPTION
**Pull request by fm3**
- [X] Backend part is done
- [X] Dataset viewing needs appropriate error message for unsupported data types
- [x] WKW lib needs to be published

### Steps to test:
- you need the changes in wkw, https://github.com/scalableminds/webknossos-wrap/pull/30 (for now, checkout that and perform `sbt publishLocal` in the `scala` folder)
- unpack the [attached test dataset](https://github.com/scalableminds/webknossos/files/2796482/signed-int-test-dataset.tar.gz) to your binaryData directory
- you should be able to import it, it should be listed as having an int8 color layer
- (viewing does not yet work, @philippotto do you want to have a look at displaying an appropriate error message?)

### Issues:
- fixes #3608 

------
- ~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~
- [x] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable
- ~[ ] Updated [documentation](../blob/master/docs) if applicable~
- [x] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
I added a validation of the layers within the front-end. Thus, the new state is:

- The back-end is capable of transmitting the different voxel types to the front-end
- The front-end will warn the user if a format is unsupported. I decided to not throw a hard error to make wk's behavior a bit more robust. One layer could be still rendered fine (if it's supported), while the other one shows artifacts.

File: app/assets/javascripts/messages.js [hunk 1]
```diff
@@ -143,6 +143,8 @@ In order to restore the current window, a reload is necessary.`,
   "dataset.import.required.zipFile": "Please select a file to upload.",
   "dataset.import.invalid_fields": "Please check that all form fields are valid.",
   "dataset.unique_layer_names": "The layer names provided by the dataset are not unique.",
+  "dataset.unsupported_element_class": (layerName: string, elementClass: string) =>
+    `The layer "${layerName}" was defined as ${elementClass}. This format is not officially supported. Please convert to the layer to a supported format.`,
```
Comment (daniel-wer) [inline app/assets/javascripts/messages.js]:
```suggestion
    `The layer "${layerName}" was defined as ${elementClass}. This format is not officially supported. Please convert the layer to a supported format.`,
```

Comment (daniel-wer) [reply app/assets/javascripts/messages.js]: We should list the supported classes somewhere (in the docs?) and link there

File: MIGRATIONS.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ User-facing changes are documented in the [changelog](CHANGELOG.md).
 
 ### Postgres Evolutions:
 - [037-add-publications.sql](conf/evolutions/037-add-publications.sql)
+- [038-more-voxel-types.sql](conf/evolutions/038-more-voxel-types.sql)
```
Comment (jstriebel) [inline MIGRATIONS.md]: @fm3 This landed in the wrong section, please move it in another PR.

Comment (fm3) [reply MIGRATIONS.md]: Good catch, I sneaked it in as part of #3720

Comment (jstriebel) [reply MIGRATIONS.md]: :+1:

**Pull request merged**